src/Stack/Init.hs

```diff
@@ -239,7 +239,7 @@ renderStackYaml p ignoredPackages dupPackages =
         [ ("user-message"     , userMsgHelp)
         , ("resolver"         , resolverHelp)
         , ("packages"         , packageHelp)
-        , ("extra-deps"       , "# Dependency packages to be pulled from upstream that are not in the resolver\n# (e.g., acme-missiles-0.3)")
+        , ("extra-deps"       , "# Dependency packages to be pulled from upstream that are not in the resolver\n# using the same syntax as packages.\n# (e.g., acme-missiles-0.3)")
```

Comment:
Hmm, not sure how I see how this is helpful.  It's kinda like "Specify packages by using the same syntax as packages".

Reply:
Yes; I didn't want to repeat `packageHelp`, though. Although this implementation (With the newline is unclear; it should probably be factored out. What do you think?

Reply:
Ahh, I see, same as the "packages field"  I get it now.